packages/vm/lib/runTx.ts

```diff
@@ -97,6 +98,23 @@ export default async function runTx(this: VM, opts: RunTxOpts): Promise<RunTxRes
   debug('-'.repeat(100))
   debug(`tx checkpoint`)
 
+  // Is it an Access List transaction?
+  if (opts.tx.transactionType === 1 && this._common.isActivatedEIP(2929)) {
```

Comment:
I think this backwards things applies to the VM as well (then `tx` in `RunTxOpts` just gets expanded by also allowing the new `2930Transaction`) and then we would just need some additional checks in places like here if the `opts.tx.transactionType` property exists before checking for the value.